arch/arm/src/sama5/sam_tsd.c

```diff
@@ -1655,7 +1735,11 @@ int sam_tsd_register(struct sam_adc_s *adc, int minor)
 
   /* Initialize the touchscreen device driver instance */
 
-  priv->adc = adc; /* Save the ADC device handle */
+  memset(priv, 0, sizeof(struct sam_tsd_s));
```

Comment:
remove memset

Reply:
Thank you for pointing all these out. Should I do the squash before or after I make these corrections? Git is not my strong point!

Reply:
squash first and then make the change. Please use --amend option after you finish:
git commit --amend